mack/__init__.py

```diff
@@ -629,7 +629,7 @@ def latest_version(delta_table: DeltaTable) -> float:
 
 
 def constraint_append(
-    delta_table: DeltaTable, append_df: DataFrame, quarantine_table: DeltaTable
+    delta_table: DeltaTable, append_df: DataFrame, quarantine_table: DeltaTable = None
```

Comment:
I actually think we should make the `quarantine_table` a required argument without a default value.  That way the user needs to explicitly instantiate the function with `quarantine_table=None` if they want to throw out the failing-constraint records.  It seems like a safer user interface.